.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
@OzzieOrca I noticed that in the ```.husky``` folder there's a ```.gitignore``` that ignores all files(using ```*```). Should this have been added? I'm making a branch for the prettier changes and just want to be sure.